.github/.stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
```

Comment:
It's possible that we will not tackle an issue within 60 days but it doesn't require any feedback, are there any other ways to mark an issue as 'leave open' besides the high and critical labels?

Reply:
Any activity on the issue (comments, labels, milestones) will push the bot back another 60 days.

Reply:
Hmm, I'm not really sure I want 'make work' type of tasks like that. But I suppose if we actually run into that issue we can always add a label for triaged and have it ignore those.